tox.ini

```diff
@@ -12,6 +12,14 @@ commands =
     sh -c 'rm -f $HOME/.matplotlib/fontList*'
     {envpython} {toxinidir}/tests.py --processes=-1 --process-timeout=300
 deps =
-    nose
+    pytest
     mock
     numpy
+
+[pytest]
+norecursedirs = .git build
+testpaths = tests
+
+[pep8]
+ignore=E111,E114,E115,E116,E121,E122,E123,E124,E125,E126,E127,E128,E129,E131,E265,E266,W503
+exclude=_delaunay.py,_image.py,_tri.py,_backend_agg.py,_tkagg.py,ft2font.py,_cntr.py,_contour.py,_png.py,_path.py,ttconv.py,_gtkagg.py,_backend_gdk.py,pyparsing*,_qhull.py,_macosx.py
```

Comment:
PEP8-check(ignoring E111 E114 E115 E116 E121 E122 E123 E124 E125 E126 E127 E128 E129 E131 E265 E266 W503)

they are now.  output log is over 4M, so I'm looking at means of making the log smaller.